### PR TITLE
Adding note about custom SCCs and restricted volume types

### PR DESCRIPTION
**Pull request by bergerhoffer**
Per @s-urbaniak.

Preview: https://deploy-preview-34161--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-bound-svc-acct-token-vol

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -1004,6 +1004,11 @@ Bound service account tokens are audience-bound and time-bound. For more informa
 
 Additionally, the kubelet refreshes tokens automatically after they reach 80% of duration, and `client-go` watches for token changes and reloads automatically. The combination of these two behaviors means that most usage of bound tokens is no different from usage of legacy tokens that never expire. Non-standard usage outside of `client-go` might cause issues.
 
+[IMPORTANT]
+====
+If you use a custom SCC that restricts volume usage in the `volumes` field, you might need to add `projected` to the list of volume types allowed by that SCC. This is because every pod is now injected with a projected volume.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-8-release-notes.adoc:1009]: @stlaz @s-urbaniak I have it as that they "might need to add projected", but is it certain enough that we can say "you must add projected"?

Comment (jeana-redhat) [inline release_notes/ocp-4-8-release-notes.adoc:1009]: This is the first use of SCC in this doc - might want to spell it out, unless it's something the audience would definitely know (I don't, but I'm not the audience :smiley:)

**Review by jeana-redhat — COMMENTED** [review]
One terminology question, but LGTM

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-8-release-notes.adoc:1009]: Good suggestion!

**Comment by bergerhoffer** [comment]
@barleyer Can you please review this?

Preview: https://deploy-preview-34161--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-bound-svc-acct-token-vol

**Comment by marun** [comment]
This PR will not be necessary after all. We've decided to fix the problem for 4.8 so that no action is required to achieve correct behavior.

4.8 bz: https://bugzilla.redhat.com/show_bug.cgi?id=1977924

**Comment by bergerhoffer** [comment]
Closing per above comment!

**Pull request closed without merging**